Doc/c-api/long.rst

```diff
@@ -89,9 +89,11 @@ All integers are implemented as "long" integer objects of arbitrary size.
    determined based on the leading characters of *str*: if *str* starts with
    ``'0x'`` or ``'0X'``, radix 16 will be used; if *str* starts with ``'0o'`` or
    ``'0O'``, radix 8 will be used; if *str* starts with ``'0b'`` or ``'0B'``,
-   radix 2 will be used; otherwise radix 10 will be used.  If *base* is not
-   ``0``, it must be between ``2`` and ``36``, inclusive.  Leading spaces are
-   ignored.  If there are no digits, :exc:`ValueError` will be raised.
+   radix 2 will be used; if *str* starts with any number other than ``0``, then
+   radix 10 will be used; otherwise :exc:`ValueError` will be raised.  If *base* is not
```

Comment:
If you send in a string representing zeroes with a base of 0, then it returns 0 because it keeps the same base (that is, keeps base 0).  I didn't know if that was special enough to include it as another option in the sentence.  The reason I put the "any number other than 0 was because I thought the error condition might need to be last.  If not, I'll reword it.